.circleci/config.yml

```diff
@@ -389,24 +389,6 @@ jobs:
 
 workflows:
   version: 2
```

Comment:
Maybe we should also delete these lines? not sure how it handles empty workflows

Reply:
the nightly flow is under workdlows right after these lines.